examples/pipelines/df_fork.py

```diff
@@ -13,18 +13,47 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
+from disdat.pipe import PipeTask
+import disdat.api as api
+from df_dup import DataMaker
+import luigi
+import pandas as pd
+
 """
 Fork Pipe
 
 Take an input DataFrame.  For each row, convert to a json string, and send to an upstream
 task specified by the string fork_task_str ('module.Class').
 
-author: Kenneth Yocum
+This examples shows:
+1.) How the input bundle shows up automatically as the parameter "pipeline_input" in upstream tasks.
+2.) How you can pass arguments when calling apply
+3.) And how an argument may be a string indicating an upstream task: '__main__.PrintRow'
+4.) How you may use **kwargs to find the dynamic set of inputs
+
+Pre Execution:
+$export PYTHONPATH=$DISDAT_HOME/disdat/examples/pipelines
+$dsdt context examples; dsdt switch examples
+$dsdt apply - DataMaker df_dup.DataMaker
```

Comment:
should this comment refer to df_fork with parameters, as done in the main below?

Reply:
Actually we run the df_dup.DataMaker pipeline to make the data bundle we will feed to df_dup.  Added an extra comment

Reply:
Thanks, Ken, I misread it, but adding more comments helps!